vms/platformvm/block/executor/proposal_block_test.go

```diff
@@ -1181,10 +1161,10 @@ func TestBanffProposalBlockDelegatorStakers(t *testing.T) {
 
 	// Case: Timestamp is after next validator start time
 	// Add a pending validator
-	pendingValidatorStartTime := defaultGenesisTime.Add(1 * time.Second)
+	pendingValidatorStartTime := genesistest.DefaultTime.Add(1 * time.Second)
 	pendingValidatorEndTime := pendingValidatorStartTime.Add(defaultMinStakingDuration)
 	nodeIDKey, _ := secp256k1.NewPrivateKey()
-	rewardAddress := nodeIDKey.PublicKey().Address()
```

Comment:
I had rationalized this as being part of the PR because almost every instance of `.PublicKey().Address()` -> `.Address()` was already included in the diff.

Do you mean specifically the instances of this that otherwise wouldn't be impacted? Or would you have preferred the full change from `.PublicKey().Address()` -> `.Address()` to be in a separate PR?

Reply:
I think I generally agree - but if the difference between `pr(cleanup A)` and `pr(cleanup all)` is small enough - I find it really temping to turn a `pr(feature A + cleanup A)` into a `pr(feature A + cleanup all)`